### PR TITLE
Added support for IP-cameras as input.

### PR DESCRIPTION
**Pull request by durgabhavaniv**

### CODE REVIEW

**Comment by durgabhavaniv** [comment]
@vladimir-dudnik I reopened new pull resuest

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/common/samples/args_helper.hpp [hunk 1]
```diff
@@ -114,9 +114,7 @@ inline std::vector<std::string> parseDevices(const std::string& device_string) {
     return devices;
 }
 
-inline std::map<std::string, uint32_t> parseValuePerDevice(const std::set<std::string>& devices,
-                                                    const std::string& values_string) {
-    //  Format: <device1>:<value1>,<device2>:<value2> or just <value>
+inline std::map<std::string, uint32_t> parseValuePerDevice(const std::set<std::string>& devices, const std::string& values_string) {
```
Comment (IRDonch) [inline demos/common/samples/args_helper.hpp]: What's the purpose of this change? You just removed a comment.

File: demos/multichannel_demo/hpe/README.md [hunk 2]
```diff
@@ -101,7 +101,11 @@ Alternatively, you can just set `-nc 3`, which simplifies application usage.
 
 If your cameras are connected to PC with indexes gap (for example, `0,1,3`), use the `-i` parameter.
 
-IP-cameras through RSTP URI interface are not supported.
+Added support for IP-cameras through RSTP URI interface.
```
Comment (IRDonch) [inline demos/multichannel_demo/hpe/README.md]: A README should document the current state, not changes. E.g. you could say "IP cameras are supported via the RTSP URI interface:".

File: demos/multichannel_demo/hpe/README.md [hunk 3]
```diff
@@ -101,7 +101,11 @@ Alternatively, you can just set `-nc 3`, which simplifies application usage.
...
+
+```
+-i   rtsp://camera_ip_add1/stream_no rtsp://camera_ip_add1/stream_no
```
Comment (IRDonch) [inline demos/multichannel_demo/hpe/README.md]: You have two identical URLs here. I think it would be more clear if they were different.

File: demos/common/samples/args_helper.hpp (hunk 1 above)
Comment (durgabhavaniv) [reply demos/common/samples/args_helper.hpp]: Yes removed comment

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/common/samples/args_helper.hpp (hunk 1 above)
Comment (IRDonch) [reply demos/common/samples/args_helper.hpp]: Ping?

**Comment by vladimir-dudnik** [comment]
@durgabhavaniv please resolve merge conflict and review question in comments

**Comment by IRDonch** [comment]
This is obsoleted by #862.
